x-pack/plugins/fleet/server/routes/agent/upgrade_handler.ts

```diff
@@ -32,7 +32,7 @@ import { checkFleetServerVersion } from '../../../common/services/check_fleet_se
 import { getAgentById } from '../../services/agents';
 
 import { getAllFleetServerAgents } from '../../collectors/get_all_fleet_server_agents';
-import { getLatestAvailableVersion } from '../../services/agents/versions';
+import { getLatestAvailableAgentVersion } from '../../services/agents/versions';
```

Comment:
I renamed this method because it's imported directly and it's not clear in context that this is the latest _agent_ version. If this was used as `agentService.getLatestAvailableVersion` maybe this wouldn't be appropriate.